src/server/scripts/Northrend/Ulduar/Ulduar/boss_freya.cpp

```diff
@@ -1159,7 +1160,7 @@ class boss_freya_summons : public CreatureScript
             if (me->GetEntry() == NPC_ANCIENT_CONSERVATOR)
             {
                 me->CastSpell(me, SPELL_HEALTHY_SPORE_SUMMON, true);
-                me->CastSpell(me, SPELL_CONSERVATOR_GRIP, true);
+				events.ScheduleEvent(EVENT_ANCIENT_CONSERVATOR_GRIP, 6000);
```

Comment:
```suggestion
                events.ScheduleEvent(EVENT_ANCIENT_CONSERVATOR_GRIP, 6000);
```
Change indent to spaces and make it equal to the rest